sdk/core/azure-core-tracing-opentelemetry/setup.py

```diff
@@ -57,9 +56,9 @@
     packages=[
         'azure.core.tracing.ext.opentelemetry_span',
     ],
-    python_requires=">=3.5.0",
+    python_requires=">=3.6.0",
```

Comment:
We are indicating that we support Python 2/2.7 in our classifiers, yet we require python>=3.6. This seem odd...

Reply:
opentelemetry requires python 3.6 or above - there is no python 2 support for the OT plugin

Reply:
Do you mean if customers are using 2.7, they are completely broken?

Reply:
customers cannot use 2.7 with opentelemtry - they should use opencencus for 3.5 and less